app/templates/views/home.html

```diff
@@ -10,7 +10,6 @@
   <div id="homepage_carousel" class="carousel slide" data-ride="carousel">
     <div class="carousel-inner">
       {% for event in all_events %}
-        {% if not event.get('past') %}
```

Comment:
Should we be showing past events in the carousel?

Reply:
Yes makes sense that we should rather than it be blank. Will put that back